base-template.sh

```diff
@@ -567,46 +665,58 @@ execute_shared_hooks() {
     # split on comma and newline
     IFS="$IFS_COMMA_NEWLINE"
 
-    # Fail if the shared root is not available (if enabled)
-    FAIL_ON_NOT_EXISTING=$(git config --get githooks.failOnNonExistingSharedHooks)
-
     for SHARED_REPO in $SHARED_REPOS_LIST; do
         unset IFS
 
         set_shared_root "$SHARED_REPO"
 
-        if [ ! -f "$SHARED_ROOT/.git/config" ]; then
-            echo "! Failed to execute shared hooks in $SHARED_REPO" >&2
-            echo "  It is not available. To fix, run:" >&2
-            echo "    \$ git hooks shared update" >&2
+        if [ "$SHARED_HOOKS_TYPE" = "--local" ] &&
```

Comment:
It might be good to suggest how to fix the issue on each of these failure return branches.
Something like, if it's forbidden remove it from the .shared file, if it doesn't exist, remove it from this config, etc.
It would be good to let people self-manage the fix for these.

Reply:
Jup, I can change that

Reply:
Done

Reply:
Done